docs-src/zdm-core/modules/migrate/pages/index.adoc

```diff
@@ -0,0 +1,92 @@
+Here is an example of these phases with ZDM using ZDM proxy, which is a simple and lightweight proxy that handles all the real-time requests generated by your client applications.
+For more, see http:addlinkhere.adoc[ZDM proxy]
+
+image::{imagesprefix}zdm-migration-all.png[Migration phases from start to finish]
+
+Before walking through illustrations of each phase, look at a pre-migration, high-level view. At this point, your client applications are performing read/write operations with an existing CQL-compatible database. That is, Apache Cassandra, DSE, or Astra DB.  
+
+image::{imagesprefix}zdm-migration-before-starting.png[Diagram shows existing CQL-compatible environment before migration starts.]
+
+Before any migration begins, complete prerequisites, prepare your environment, and set up the recommended infrastructure.
+////
+
+== Migrations with and without zero downtime examples
+
+Data migrations with downtime are those where the data being migrated is critical for ongoing operations, and the migration process itself requires systems to be taken offline temporarily. Downtime may be required for:
+
+* Migration for large-scale systems: these systems may require downtime for completely smooth transition.
+* Database migration: large database migrations or database software upgrades may need downtime for data consistency and integrity.
+* Storage system migration: migrating from on-prem to cloud may need downtime to reconfigure systems.
+* Network infrastructure changes: these systems may require downtime for completely smooth transition.
+
+A migration with no downtime is seamless and ideal, but doesn't always happen or a viable option. 
+Downtime may not be required for:
+
+* Continuous data replication: When real-time data replication is implemented, changes made to the source data are continuously mirrored to the target system. 
+This means both systems remain synchronized without the need for downtime. 
+* Online database migration: Some database migration allows for the migration to happen while the database remains online and accessible. Technologies like database replication and synchronization tools can also facilitate seamless migration with minimal disruption.
+* Incremental data migration:Transfers data in smaller batches over time, which allows for minimal disruption to ongoing operations since only a portion of the data is being migrated at any given time. 
+This option can be used for large-scale migrations where downtime is not feasible or acceptable.
+* Load balancing and redundancy: In environments with redundant systems or load balancing capabilities, data migrations can be performed without downtime by routing traffic to the target system while the migration is in progress. This allows users to access services without interruption, as the load is distributed between the source and target systems.
+* Cloud-based data migration: This often provide tools and services that enable live data migration with minimal downtime. 
+
```

Comment:
The primary criterion for zero downtime migration is whether the business can tolerate a brief outage. Migration scenarios are documented here [Migration Scenarios](https://docs.datastax.com/en/astra/data-migration/introduction.html#migration-scenarios)